e2e-tests/src/tests/constraints-load-old-sim-results.test.ts

```diff
@@ -0,0 +1,111 @@
+import { expect, test } from '@playwright/test';
+import req, { awaitSimulation } from '../utilities/requests.js';
+
+test.describe.serial('Temporal Subset Simulation', () => {
```

Comment:
Looks like we need a new title for this one. Perhaps "Check Constraints Against Given Simulation Dataset"?